packages/twenty-server/src/metadata/field-metadata/field-metadata.service.ts

```diff
@@ -228,9 +228,23 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
           )
         : fieldMetadataInput;
 
-    const updatedFieldMetadata = await super.updateOne(id, updatableFieldInput);
+    const updatedFieldMetadata = await super.updateOne(id, {
```

Comment:
it looks to me that this allows the update of standard fields too:
In buildUpdatableStandardFieldInput, the name, desc, icon are not part of fieldMetadataInputOverrided

Reply:
Hummm I think you're right, I've fixed it, sorry for that !

Reply:
I think your fix still allows to update the name of a standard field